ods_ci/tasks/Resources/Provisioning/Hive/provision.robot

```diff
@@ -209,18 +207,9 @@ Login To Cluster
     Log    ${result.stdout}\n${result.stderr}     console=True
     Should Be True    ${result.rc} == 0
 
-Set Cluster Storage
```

Comment:
We don't need this anymore? (I have no knowledge, just don't understand why we don't need it now) :slightly_smiling_face:

Reply:
Okay then, hope it won't break anything. Thank you for explanation! :slightly_smiling_face: